src/Aspire.Dashboard/Components/Controls/ResourceDetails.razor.cs

```diff
@@ -74,6 +74,7 @@ protected override void OnInitialized()
             new KnownProperty(KnownProperties.Resource.DisplayName, Loc[Resources.Resources.ResourcesDetailsDisplayNameProperty]),
             new KnownProperty(KnownProperties.Resource.State, Loc[Resources.Resources.ResourcesDetailsStateProperty]),
             new KnownProperty(KnownProperties.Resource.CreateTime, Loc[Resources.Resources.ResourcesDetailsStartTimeProperty]),
+            new KnownProperty(KnownProperties.Resource.StopTime, Loc[Resources.Resources.ResourcesDetailsStopTimeProperty]),
```

Comment:
Should we display create and startup time in the properties, or would that just be confusing?

I think they would only display different values if we support starting a docker container without recreating it.

Reply:
No we don't care about the creation time